siwe/siwe.py

```diff
@@ -7,11 +7,17 @@
 from typing import Any, Dict, Iterable, List, Optional, Union
 
 import eth_utils
-from dateutil.parser import isoparse
```

Comment:
if `isoparse` is no longer used, I think you can also remove `python-dateutil` from `pyproject.toml`

Reply:
deptry will complain about that in the CI if it an issue